Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+.container {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  margin: 30px 0 90px 120px;
+  width: 1000px;
```

Comment:
this will break your layout if your screen is smaller than 1000px. use:
```
width: 100%;
max-width: 1000px;
```
to contemplate smaller screens.